README.md

```diff
@@ -12,6 +12,23 @@ Main motivation for Intl.DisplayNames project was to enable developers to get tr
 * Reduce translation cost for developers.
 * Consistent translation of language, region and script display name on the web.
 
+### Scope
+This proposal is not intended to provide a translation for all strings, but rather focus on particular items which are applcation-independent. There are two classes of strings we're considering here:
+
+- Strings that are already available because they're needed for other APIs. For example, names for the days of the week are necessary to provide Intl.DateTimeFormat capabilities. In the absense of a direct API, we see programs parsing the output of Intl.DateTimeFormat to find these names, which is an unreliable/unstable technique.
```

Comment:
```suggestion
- Strings that are already available because they're needed for other APIs. For example, names for the days of the week are necessary to provide Intl.DateTimeFormat capabilities. In the absence of a direct API, we see programs parsing the output of Intl.DateTimeFormat to find these names, which is an unreliable/unstable technique.
```